governance/votes/Executive vote - July 16, 2021.md

```diff
@@ -0,0 +1,100 @@
+| GovAlpha (GOV-001) | 273,334 DAI|
+| Real World Finance (RWF-001) | 155,000 DAI|
+| Risk (RISK-001) | 182,000 DAI|
+| Protocol Engineering (PE-001) | 510,000 DAI|
+| Oracles (ORA-001) | 419,677 DAI|
+| Oracles (ORA-001) - Continuity Reserve | 800,000 DAI|
+| **Total** | **2,510,503 DAI** |
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Decrease the Bid Duration (ttl) for FLAP Auctions
+
+As per the successful outcome to [this](https://vote.makerdao.com/polling/Qmb65Ynh?network=mainnet#poll-detail) governance poll, the Bid Duration (ttl) for FLAP auctions will be **decreased** from 60 minutes to **30 minutes**, if this executive proposal passes.
+
+More information on Surplus (FLAP) Auctions and their parameters can be found [here](https://docs.makerdao.com/auctions/the-auctions-of-the-maker-protocol#surplus-auction). Please see this [signal request](https://forum.makerdao.com/t/signal-request-adjusting-ttl-bid-duration-on-flap-auctions/8673) for further details and discussion on this parameter change.
+
+### Increase the RWA-02 (NS-DROP) Debt Ceiling
+
+As per the successful outcome to [this](https://vote.makerdao.com/polling/QmUNouQ7?network=mainnet#poll-detail) governance poll, the RWA-02 (NS-DROP) [Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling) will be **increased** from 5 million DAI to **20 million DAI**, if this executive proposal passes.
+
+Please see this [signal request](https://forum.makerdao.com/t/signal-request-increase-debt-ceiling-of-rwa-02-ns-drop-from-5m-to-20m/9077) for further details and discussion on this parameter change.
+
+### July Core Unit Budget Distributions
+
+Provided the [previous executive vote](https://vote.makerdao.com/executive/template-executive-vote-rates-changes-july-core-unit-budgets-and-psm-dc-iam-july-2-2021?network=mainnet#proposal-detail) does not pass before this executive does, the following Core Unit budget distributions will occur. In the event the previous executive proposal does pass before this executive, these distributions will not occur.
+
+**Note:** In the event that **both** executive proposals pass **and** this executive passes before the previous one, the Core Units will either return the excess funds or have the excess amount subtracted from future distributions.
```

Comment:
'**Note:** In the event that **both** executive proposals pass **and** this executive passes before the previous one' -> '**Note:** In the event that the 2021-07-02 executive passes **after** this one, the July budget distributions will be duplicated and Core Units will either...'

Maybe?